third_party/move/move-compiler-v2/tests/checking-lang-v2.2/lambda/constraints.exp

```diff
@@ -18,7 +18,7 @@ error: phantom type `T` can only be used as an argument for another phantom type
 4 │     struct Func3<phantom T>(|u64|(T, T)) has copy, drop;
   │                                   ^
 
-error: tuple type `(u64, u64)` is not allowed as a type argument
+error: tuple type `(u64, u64)` is not allowed
```

Comment:
@rahxephon89 

> what is the difference between None and default?

Note that in this error message, "as a type argument" does not make sense. This is because in the method `TypeUnificationError::message_with_hints_and_labels`, for the case `TypeUnificationError::ConstraintUnsatisfied`, if the `ctx_opt` is `None`, instead of the default, we use the text "as a type argument".